web/styles/modal.css

```diff
@@ -429,8 +429,8 @@
     width: fit-content;
 }
 
-.dropdown-widget-button {
-    width: 206px;
+.overlay-content .dropdown-widget-button {
```

Comment:
This might change width in some of the places. One place that I noticed is the filter in "Inbox view".

Reply:
Also, the dropdown menu opened on clicking these buttons in smaller than the button. Not a major issue and it looks fine, but mentioning in case we want to increase the menu's width as well.

Reply:
Ah dang so `dropdown-widget-button` styling from `modal.css` was being used in non-modal places... thanks for noticing that. I'll add the 206px width to the style rules in `app_components.css`

Reply:
I'm not noticing the dropdown being more narrow than the button in modals:

<img width="847" alt="image" src="https://github.com/user-attachments/assets/d1f717c9-933f-41f8-ab46-0d42c60a11fa" />

Can you give me an example of a place that's doing that? I think increasing the width would be good if that's still happening.

Reply:
The change here is not restricted to modals. The dropdown menu is wider than button in "Organization settings" section for stream settings like "New channel announcements".

Reply:
> The change here is not restricted to modals. 

What do you mean? Do you see modal and overlay as different, or is this change showing up in some non-modal non-overlay place somehow?

Reply:
Yes, I was thinking modals and overlays as being different, but I guess you are considering all of them as modals. Anyways, I am not seeing this in non-modal, non-overlay places.

Reply:
Ok cool. So my plan is to change this in all overlay/modal screens and nowhere else and have been manually testing both.

It looks like the dropdown is a bit different width on `main` at 14px, but fairly close. I don't see a super easy way to give the dropdown the correct width for all dropdowns in modals because it's being rendered with tippy, but we an do a followup for that if it feels important.

<img width="230" alt="image" src="https://github.com/user-attachments/assets/f7603607-b799-4d54-9a55-dbc7a7531d0d" />

Reply:
Sounds good.